lib/screens/explore_screens/vouch/sponsor_tab/mappers/load_sponsors_mapper.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:seeds/datasource/remote/model/member_model.dart';
+import 'package:seeds/domain-shared/page_state.dart';
+import 'package:seeds/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/screens/explore_screens/vouch/sponsor_tab/interactor/viewmodels/sponsor_bloc.dart';
+
+class LoadSponsorsStateMapper extends StateMapper {
+  SponsorState mapResultToState(SponsorState currentState, Result<List<MemberModel>> result) {
+    if (result.isError) {
+      return currentState.copyWith(pageState: PageState.failure, errorMessage: result.asError!.error.toString());
+    } else {
+
+      return currentState.copyWith(
+        pageState: PageState.success,
+        sponsors: result.asValue!.value
+
+      );
```

Comment:
Horrible la formatt config de Android Studio.

Reply:
this is him, not android studio 
pls fix

Reply:
fix